install.rst

```diff
@@ -0,0 +1,114 @@
+Install an OMERO server
+=======================
+
+This chapter describes an installation of an OMERO.server and the most important post-installation steps, such as additional apps installation and LDAP configuration. The installation of the server on local machine is not recommended, except for testing or development purposes. The workflow is aimed at system administrators intending to familiarize themselves with OMERO installation.
+
+Description
+-----------
+
+We will show here:
+
+
+-  Where to find the OMERO.server requirements.
+
+-  How to install OMERO.server quickly using a simple Ansible playbook example.
+
+-  How to install OMERO.server very quickly using a provided Docker image.
+
+-  How to configure OMERO.server to work with LDAP, both using Ansible or using manual configuration.
+
+-  How to install applications for OMERO.web, such as OMERO.figure.
+
+Resources
+---------
+
+- Webpage  `Start with OMERO at your Institution <https://www.openmicroscopy.org/omero/institution/getting-started.html>`_
+- `Ansible <https://www.ansible.com/>`_
+- `Ansible documentation <https://docs.ansible.com/ansible_community.html>`_
+- `Example <https://github.com/ome/omero-deployment-examples>`_ and `production <https://github.com/ome/prod-playbooks>`_ ansible playbooks.
+- `OMERO installation workshop presentation <https://downloads.openmicroscopy.org/presentations/2020/Dundee/Workshops/OME2020-OMERO-Installation/#/>`_
+- `Manual installation documentation <https://omero.readthedocs.io/en/v5.6.5/sysadmins/unix/server-centos7-ice36.html>`_.
+- `System requirements <https://omero.readthedocs.io/en/v5.6.5/sysadmins/system-requirements.html>`_
+- `List of Apps for OMERO.web and Command line plugins <https://www.openmicroscopy.org/omero/apps/>`_
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
+
+Ansible
+-------
+
+For OMERO installation, we recommend to use the Ansible software suite which enables infrastructure as a code.
+
+Install ansible (at the time of writing, the ansible-core version was 2.11.12, python version was 3.6.8). Other versions might work too, but are not routinely tested.::
```

Comment:
Python 3.6 is EOL. So we should test with newer Python version. For CentOS 7 this means installing scl Python 3.8.

Reply:
okay, I will try

Reply:
> For CentOS 7 this means installing scl Python 3.8.

I remember now (as I repeated it) why I did not mention Python 3.8. I tried, and it fails at the installation of OMERO step.  Also, installing Python 3.8 seems very difficult on CentOS 7. Could you point me to what you mean by the ``scl`` please ?

Once I installed Python 3.8.12 from source

```
yum install gcc openssl-devel bzip2-devel libffi-devel zlib-devel
wget https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tgz
yum install wget
wget https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tgz
tar xzf Python-3.8.12.tgz
cd Python-3.8.12
./configure --enable-optimizations
yum install make
cd /Python-3.8.12
make install
```
and then installed pip and ansible as below, the ``ansible-playbook`` command in that env is failing as below. So for me, the python 3.8 workflow just does not work. I assumed (probably erroneously) that this is because OMERO is using python 3.6. 

```
pip3 install --upgrade pip
pip install ansible
ansible --version
ansible [core 2.13.7]
python3 --version
Python 3.8.12
pip --version
pip 22.3.1 from /usr/local/lib/python3.8/site-packages/pip (python 3.8)

...
ansible-playbook --become --connection=local -i hosts.yml playbook.yml
....
TASK [ome.omero_server : omero server | install requirements] ******************
fatal: [localhost]: FAILED! => {"changed": false, "cmd": ["/opt/omero/server/venv3/bin/pip", "install", "omego==0.7.0", "https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl", "omero-py>=5.6.0", "omero-dropbox>=5.6.1", "jinja2", "omero-server[default]==0.1.0"], "msg": "\n:stderr: ERROR: zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl is not a supported wheel on this platform.\nWARNING: You are using pip version 21.1.1; however, version 22.3.1 is available.\nYou should consider upgrading via the '/opt/omero/server/venv3/bin/python3 -m pip install --upgrade pip' command.\n"}
```

My workflow with python 3.6 is much easier, see https://github.com/ome/ansible-example-omero-onenode/issues/13 but mainly it works to the very end (i.e. OMERO starts and works), whereas the python 3.8 simply does not install OMERO. Happy for any suggestions, but otherwise blocked on python 3.8 - does not seem to work in my hands.

Reply:
cc @jburel

Reply:
@jburel Aha, maybe https://serverfault.com/questions/1114100/no-python38-pkg-found-for-centos-7 - this workflow seems easier with respect to installing python 3.8, but this is not the real problem I am facing - I have successfully installed python 3.8.12, but the followup OMERO install is failing - this is the main issue.

Reply:
On Ubuntu 20.04 with python 3.8 installed and ansible run remotely from my local mac, I am getting a similar error as on Cent OS 7 with python 3.8:

```
TASK [ome.omero_server : omero server | install requirements] *********************************************************************
fatal: [romantic_easley]: FAILED! => {"changed": false, "cmd": ["/opt/omero/server/venv3/bin/pip3", "install", "omego==0.7.0", "https://github.com/ome/zeroc-ice-ubuntu2004/releases/download/0.2.0/zeroc_ice-3.6.5-cp38-cp38-linux_x86_64.whl", "omero-py>=5.6.0", "omero-dropbox>=5.6.1", "jinja2", "omero-server[default]==0.1.0"], "msg": "\n:stderr: ERROR: zeroc_ice-3.6.5-cp38-cp38-linux_x86_64.whl is not a supported wheel on this platform.\n"}
```

Again, even on different OS and different Ansible version, the same error in same place - I think it is clearly the python 3.8 which is the "culprit" here.

Reply:
Further to this ``python 3.8`` saga: I think I can report:

- on Ubuntu 20.04 with python 3.8, as well as on CentOS 7 with python 3.8
- on Ubuntu 18.04 (which comes with python 3.6) 

I get the error described in https://github.com/ome/omero-guides/pull/236/files#r1051070068 which actually resembles https://forum.image.sc/t/installing-omero-py-on-python-3-7/44128 discussion - any insights @joshmoore please ? (note that I cannot easily create a conda env etc., as I am simply running our playbooks with default parameters) - any easy (Ansible) way out of this ? Ta